cmds/tfuser/cmds_vm.go

```diff
@@ -0,0 +1,63 @@
+package main
```

Comment:
tfuser is not maintained actually.

I would revert changes here actually, since this is probably not working against this version of the explorer.